### DISCUSSION TITLE
版本3.7中使用linechart中y轴使用MinMaxAuto坐标轴混乱

### DISCUSSION THREAD

**Discussion by tianyu-x1a** [XCharts-Team/XCharts#262] [Q&A]
<img width="170" alt="image" src="https://github.com/XCharts-Team/XCharts/assets/84573792/e2c7c203-1795-41fd-9af3-50acc33893cb">
左侧坐标轴变成332211，右侧数字和左侧坐标轴对不上。

**Comment by monitor1394** [comment]
是不是y轴限制只显示整数了？

**Comment by tianyu-x1a** [comment]
是这样啊。显示小数的表看上去很混乱，还是需要效果在整数y轴中实现。



在 2023-07-08 14:14:49，"monitor1394" ***@***.***> 写道：

是不是y轴限制只显示整数了？

**Reply by monitor1394** [reply]
就是问你有没有设置了y轴文本的显示格式，截图发一下Y轴的AxisLbel下的配置看看。
如果没有设置，也可以自己指定显示的小数位数，如将AxisLabel的numericFormater参数设置为f1.

**Comment by monitor1394** [comment]
v3.8.0已优化默认设置。
